src/screens/HomeScreen.js

```diff
@@ -158,24 +170,25 @@ export const HomeScreen = ({ navigation }) => (
           return (
             <View>
               <CardList navigation={navigation} data={pointsOfInterest} horizontal />
-              <TextList
-                navigation={navigation}
-                data={[
-                  {
-                    id: '-1',
-                    title: 'Alle Orte und Touren anzeigen',
-                    routeName: 'Index',
-                    params: {
-                      title: 'Orte und Touren',
-                      query: 'pointsOfInterest',
-                      queryVariables: {},
-                      rootRouteName: 'PointsOfInterest'
-                    },
-                    bottomDivider: false,
-                    topDivider: true
-                  }
-                ]}
-              />
+
+              {!!pointsOfInterest && (
```

Comment:
we can get rid of the check for `!!pointsOfInterest` here when checking before with an early `return null`, like for news items and event records. than it would be is safe to render `<Wrapper><Button...` directly.